modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -19,8 +19,21 @@ IMPORTANT: OpenShift oc executable must be located in a system location that is
 +
 IMPORTANT: Ensure that you can run link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up] with no errors before moving on to the installation of AeroGear.
 +
-
+* Docker (version 1.22 or above)
++
 * A link:https://hub.docker.com/[Docker Hub] account
+** For Linux (RHEL), manage Docker as a non-root user:
```

Comment:
Tested locally and this works

Reply:
what do you mean

Reply:
The new bash commands below - I used them and things worked

Reply:
we'd removed non-root user docker material before, need to follow up with eng about this